.github/content/code-review-checklist.md

```diff
@@ -0,0 +1,23 @@
+[changelog guide]: https://docs.plasmapy.org/en/latest/contributing/changelog_guide.html
+[coding guide]: https://docs.plasmapy.org/en/latest/contributing/coding_guide.html
+[doc guide]: https://docs.plasmapy.org/en/latest/contributing/doc_guide.html
+[testing guide]: https://docs.plasmapy.org/en/latest/contributing/testing_guide.html
+
+Thank you for contributing to PlasmaPy! The project's future depends deeply on contributors like you, so we deeply appreciate it! 🌱 The following checklist will be used by the code reviewer to help guide the code review process.
```

Comment:
I would love it if there were more emojis in this file. But I'm not sure which would be best.